### ISSUE TITLE
Won't work with custom cfw 6.61 on both PRO-C and LME

### ISSUE THREAD

**Issue by MojoJojoDojo** [unknownbrackets/maxcso#52]
As the title say.
Using the latest maxcso version here.
Sometimes very rarely they do work but most of the time they either freeze the game or make it crash.

**Comment by unknownbrackets** [comment]
Interesting, that's a bit unexpected.  I just tried with a game and can reproduce.

If you use `--no-libdeflate` does it work?  You can do this with a batch file.  Create a file in Notepad and save as "maxcso-psp.cmd" (include the quotes) in the same folder as maxcso.  Use this content:

```cmd
"%~dp0\maxcso.exe" --no-libdeflate %*
```

Then you should be able to drag and drop the same way and try on a PSP.

If you can confirm this also works around the issue for you, I'll have to disable libdeflate by default since I want the default output to work on a PSP.  Interesting that it doesn't - standard zlib can normally decompress libdeflate output, so this is likely a CFW bug.

-[Unknown]

**Comment by MojoJojoDojo** [comment]
Yeah that seems to do the job with at least 1 game I tried (Toaru Majutsu no Index).
TBH it's not that bad because I try to compress/dump games as PBPs when I can and it saves more space.
But still would be important for other users etc that dont get why they get that issue.
